### PR TITLE
PromoCode and payments

### PR DESCRIPTION
**Pull request by peterpolman**

### CODE REVIEW

File: src/controllers/promo_codes/post.docs.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/**
```
Comment (peterpolman) [inline src/controllers/promo_codes/post.docs.ts:1]: @bram-rongen Separate file for swagger docs, suggested `*.docs.ts` as a filename but open to alternatives:) 

`*.spec.ts` is usually meant for tests right? The name kinda aligns with what this stuff is meant for, but hesitant to use it.

Comment (bram-rongen) [reply src/controllers/promo_codes/post.docs.ts:1]: docs seems better indeed!

File: scripts/deploy/index.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ async function deploy(network: NetworkProvider.Test | NetworkProvider.Main) {
     console.log('*** UPDATE YOUR .ENV ***');
 }
 
-const arg = process.argv[2];
```
Comment (bram-rongen) [inline scripts/deploy/index.ts:20]: This worked fine, please revert change :)

Comment (peterpolman) [reply scripts/deploy/index.ts:20]: Ah that was a little local hack:P

**Pull request merged**